src/components/NewRequest.jsx

```diff
@@ -78,12 +78,11 @@ const NewRequest = () => {
               required
             />
             <Form.Input>
-              <label>Category</label>
               <Dropdown
                 selection
                 id="category"
                 name="category"
-                placeholder="Other"
+                placeholder="Category"
```

Comment:
What happens if nothing is selected and this is sent off to the backend?

Reply:
good spot, i have made it required now so they have to choose a category